tox.ini

```diff
@@ -1,13 +1,11 @@
-[tox]
-envlist = py27, py27-scrapy1.0.x, py35, py36
-
-[testenv]
-deps =
-    -r{toxinidir}/requirements.txt
-    -r{toxinidir}/requirements-dev.txt
-commands = py.test {posargs}
-
-[testenv:py27-scrapy1.0.x]
-deps =
-    Scrapy<1.1
-    -r{toxinidir}/requirements-dev.txt
+[tox]
+envlist = py27, py27-scrapy{1.0,1.1,1.2,1.3}, py35, py36, {py35,py36}-scrapy{1.1,1.2,1.3}
```

Comment:
anyway your change means that we will need to update tests with every minor Scrapy release, why keep duplicate env? if you want to keep `py*` envs let's maybe remove `py*-scrapy1.3` envs and add them once you release 1.4? `py*` envs will test latest stable version in this case and `py*-scrapy*` will test older minor releases.